src/pages/access_review/AccessReview.js

```diff
@@ -33,15 +33,15 @@ class AccessReview extends React.PureComponent {
   }
 
   async darReviewAccess() {
-    const { darId, voteId, rpVoteId } = this.props.match.params;
+    const { darId, voteId } = this.props.match.params;
 
     // Access Election Information
     const getElectionInformaion = async(darId, voteId) => {
       try{
         const accessVote = await Votes.getDarVote(darId, voteId);
         const accessElectionReview = await Election.findDataAccessElectionReview(accessVote.electionId, false);
         const accessElection = fp.isNil(accessElectionReview) ? null : accessElectionReview.election;
-        const rpElectionReview = fp.isNil(rpVoteId) ? null : await Election.findRPElectionReview(accessElection.electionId, false);
+        const rpElectionReview = fp.isNil(accessElection) ? null : await Election.findRPElectionReview(accessElection.electionId, false);
```

Comment:
This is the meat of the PR. Since we don't really need the `rpVoteId` to find the RP election, we can instead just query for it and populate the RP election if it exists.